contrib/completion/zsh/_docker

```diff
@@ -1123,9 +1123,8 @@ __docker_subcommand() {
         (search)
             _arguments $(__docker_arguments) \
                 $opts_help \
-                "($help)--automated[Only show automated builds]" \
+                "($help)*"{-f=,--filter=}"[Filter values]:filter: " \
                 "($help)--no-trunc[Do not truncate output]" \
-                "($help -s --stars)"{-s=,--stars=}"[Only display with at least X stars]:stars:(0 10 100 1000)" \
                 "($help -):term: " && ret=0
             ;;
```

Comment:
@sdurrheimer thank you very much on that !! :heart_eyes_cat: 
I wasn't sure I should add them or not, and even more if I could write them :stuck_out_tongue:

By the way, I couldn't find the filter completion on `docker images`, is that normal ?